src_assets/common/assets/web/configs/tabs/encoders/VAAPIEncoder.vue

```diff
@@ -0,0 +1,28 @@
+<script setup>
+import { ref } from 'vue'
+
+const props = defineProps([
+  'platform',
+  'config',
+])
+
+const config = ref(props.config)
+</script>
+
+<template>
+  <div id="vaapi-encoder" class="config-page">
+    <!-- Strict RC Buffer -->
+    <div class="mb-3">
+      <label for="vaapi_strict_rc_buffer" class="form-label">{{ $t('config.vaapi_strict_rc_buffer') }}</label>
+      <select id="vaapi_strict_rc_buffer" class="form-select" v-model="config.vaapi_strict_rc_buffer">
+        <option value="enabled">{{ $t('_common.enabled') }}</option>
+        <option value="disabled">{{ $t('_common.disabled') }}</option>
```

Comment:
```suggestion
        <option value="disabled">{{ $t('_common.disabled_def') }}</option>
```

This will have slightly different text for the default option.

Reply:
Good catch, I think I forgot to pull before I rebased and amended the commit.